lib/FeatureSetUtils/Utils/FeatureSetBuilder.py

```diff
@@ -42,25 +42,22 @@ def _validate_upload_featureset_from_diff_expr_params(self, params):
 
         # check for required parameters
         for p in ['diff_expression_ref', 'workspace_name',
-                  'p_cutoff', 'q_cutoff', 'fold_scale_type', 'fold_change_cutoff']:
+                  'p_cutoff', 'q_cutoff', 'fold_change_cutoff']:
             if p not in params:
                 raise ValueError('"{}" parameter is required, but missing'.format(p))
 
-        fold_scale_type = params.get('fold_scale_type')
-        validate_fold_scale_type = ['linear', 'logarithm']
-        if fold_scale_type not in validate_fold_scale_type:
-            error_msg = 'Input fold scale type value [{}] is not valid'.format(fold_scale_type)
-            raise ValueError(error_msg)
+        if params.get('fold_scale_type'):
```

Comment:
Allow the parameter but require that it be log

Reply:
okay.  I'll put in the check there but not refer to it anywhere in the code at this point.